execute-test-cases.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Execute your tests
+description: "Run your tests to find bugs in your app. We will help you figure out what went wrong."
+icon: "play"
+---
+
+## Run your active test cases
+
+You should have number of **active tests** in your test suite by now. They are `on`.
+
+To check if there are any bugs in your new build or in production preventing your users from operating your app, you will have to **run the tests**.
+
+### 1. Trigger a test run
+
+To run all active tests instantly click `run all` and then `in app`. This will trigger a test run resulting in a [test report](/test-reports.mdx) visible in the Octomind app. We run the tests in parallel to make sure they run completes under 20 minutes, regadless of the number of your active test cases.
```

Comment:
```suggestion
To run all active tests instantly click `run all` and then `in app`. This will trigger a test run resulting in a [test report](/test-reports.mdx) visible in the Octomind app. We run the tests in parallel to make sure they run completes under 20 minutes, regardless of the number of your active test cases.
```